conbench/entities/benchmark_result.py

```diff
@@ -775,8 +775,8 @@ def validate_and_aggregate_samples(stats_usergiven: Any):
             "q1": q1,
             "q3": q3,
             "median": float(np.median(samples)),
-            "min": np.min(samples),
-            "max": np.max(samples),
+            "min": float(np.min(samples)),
+            "max": float(np.max(samples)),
```

Comment:
These `float()` changes are unrelated to the rest of this PR. When I tested locally, I was getting some test errors that sqlalchemy was submitting SQL to the DB that looked something like
```sql
INSERT INTO benchmark_result (..., median, min, ...)
VALUES (..., 5.3, np.float64(5), ...)
```
This obviously isn't valid SQL. It's been a while since we developed in this repo! 🙂 I guess some dependency was updated (probably sqlalchemy). So this fixes that problem.